app/scripts/lib/bloomfilter.js

```diff
@@ -0,0 +1,118 @@
+define([
+],
+function() {
+  'use strict';
+  var typedArrays = typeof ArrayBuffer !== "undefined";
```

Comment:
The bloom filter-data file here is for passwords, so if we are not going to use the filter for anything other than fxa, then it should be fxa specific. What do you think @vladikoff ?